time-elements.js

```diff
@@ -184,6 +192,66 @@
     }
   };
 
+  RelativeTime.prototype.timeUntil = function() {
+    var ms = this.date.getTime() - (new Date().getTime());
+    return this.timeUntilFromMs(ms);
+  };
+
+  RelativeTime.prototype.timeUntilFromMs = function(ms) {
+    var sec = Math.round(ms / 1000);
+    var min = Math.round(sec / 60);
+    var hr = Math.round(min / 60);
+    var day = Math.round(hr / 24);
+    var month = Math.round(day / 30);
+    var year = Math.round(month / 12);
+    if (month >= 18) {
+      return year + ' years from now';
+    } else if (month >= 12) {
+      return 'a year from now';
+    } else if (day >= 45) {
+      return month + ' months from now';
+    } else if (day >= 30) {
+      return 'a month from now';
+    } else if (hr >= 36) {
+      return day + ' days from now';
+    } else if (hr >= 24) {
+      return 'a day from now';
+    } else if (min >= 90) {
+      return hr + ' hours from now';
+    } else if (min >= 45) {
+      return 'an hour from now';
+    } else if (sec >= 90) {
+      return min + ' minutes from now';
+    } else if (sec >= 45) {
+      return 'a minute from now';
+    } else if (sec >= 10) {
+      return sec + ' seconds from now';
+    } else {
+      return 'just now';
+    }
+  };
+
+  RelativeTime.prototype.microTimeUntil = function() {
+    var ms = this.date.getTime() - (new Date().getTime());
+    var sec = ms / 1000;
+    var min = sec / 60;
+    var hr = min / 60;
+    var day = hr / 24;
+    var month = day / 30;
+    var year = month / 12;
```

Comment:
Maybe @dgraham remembers why the micro format works different https://github.com/github/time-elements/pull/32/files#r69352974